website/docs/getting-started/setup/upgrade-to-business-edition/docker.mdx

```diff
@@ -0,0 +1,129 @@
+---
+description: Follow the steps to upgrade the Appsmith community docker installation to a Business Edition.
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Docker
+
+Follow the steps listed below to **upgrade** your Community Edition (CE) to a Business Edition (EE) installed on docker.
+
+### Step 1: Database backup
```

Comment:
Checked with Tanmay. He said as it is already there, leave it as is in case these links have been shared to avoid redirect URLs. But going forward, we should make sure headings are consistent throughout.